src/openms/include/OpenMS/CONCEPT/Constants.h

```diff
@@ -443,6 +443,13 @@ namespace OpenMS
       inline const std::string   XFDR_FDR = "XFDR:FDR";
     }
 
+    namespace Versions
```

Comment:
@timosachsenberg Not sure if that is what you had in mind.

Reply:
hmm I would put it in a file related to sirius. I just meant instead of static constexpr